javascript/javascript3/week2/Exercises.js

```diff
@@ -0,0 +1,113 @@
+async function getMovies() {
+  try {
+    const response = await fetch(
+      "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    );
+    const movies = await response.json();
+    console.log(movies);
```

Comment:
Nitpick(logging): While tracing is very helpful when building the code, it's usually a good idea to remove them before committing your code!